docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -212,25 +217,150 @@ Iceberg. In addition to the :ref:`globally available <sql-globally-available>`
 and :ref:`read operation <sql-read-operations>` statements, the connector
 supports the following features:
 
-* :doc:`/sql/insert`
-* :doc:`/sql/delete`, see also :ref:`iceberg-delete`
-* :doc:`/sql/update`
-* :doc:`/sql/merge`
-* :ref:`sql-schema-table-management`, see also :ref:`iceberg-tables`
-* :ref:`sql-materialized-view-management`, see also
-  :ref:`iceberg-materialized-views`
-* :ref:`sql-view-management`
+* :ref:`sql-write-operations`:
+
+  * :ref:`sql-schema-table-management`, see also :ref:`iceberg-schema-table-management` and :ref:`iceberg-tables`
```

Comment:
Generated page has repeated "Schema and table management". By the way, why do we want both `sql-schema-table-management` & `iceberg-schema-table-management`? 

<img width="701" alt="Screen Shot 2022-10-06 at 14 35 42" src="https://user-images.githubusercontent.com/6237050/194222601-74b1b28a-e635-44fe-999e-8eb964de2d61.png">

Reply:
We don't need both. Actually having the links on the same line is misleading.
The `iceberg-schema-table-management` has anyway a link towards `sql-schema-table-management` so taking out the `sql-schema-table-management` link from this listing probably ok.